frontend/src/components/dashboard/DashboardHeader.tsx

```diff
@@ -69,168 +60,135 @@ export default class extends React.Component<State> {
-                      {"Learn more about the key concepts"}
-                    </a>
-                  </p>
-                </Term>
-              }
-              imgLink="/static/images/icon-m-height.svg"
-              text={details ? details.lastBlockHeight.toLocaleString() : ""}
-              loading={loading}
-            />
-          </Col>
-          <Col xs="12" md="2">
-            <CardCell
-              title={
-                <Term title={"TXs"}>
-                  {"The number of transactions since genesis. "}
-                  <a
-                    href={
-                      "https://docs.nearprotocol.com/docs/concepts/transaction"
-                    }
-                  >
-                    docs
-                  </a>
-                </Term>
-              }
-              imgLink="/static/images/icon-m-transaction.svg"
-              text={details ? details.totalTxCount.toLocaleString() : ""}
-              loading={loading}
-            />
-          </Col>
-          <Col xs="12" md="2">
-            <CardCell
-              title={
-                <Term title={"TPD"}>
-                  {"The number of transactions in the last 24 hours. "}
-                  <a
-                    href={
-                      "https://docs.nearprotocol.com/docs/concepts/transaction"
-                    }
-                  >
-                    docs
-                  </a>
-                </Term>
-              }
-              imgLink="/static/images/icon-m-transaction.svg"
-              text={details ? details.lastDayTxCount.toLocaleString() : ""}
-              loading={loading}
-            />
-          </Col>
-          <Col xs="12" md="2">
-            <CardCell
-              title={
-                <Term title={"Accounts"}>
-                  {"Total number of accounts created on this net. "}
-                  <a
-                    href={"https://docs.nearprotocol.com/docs/concepts/account"}
-                  >
-                    docs
-                  </a>
-                </Term>
-              }
-              imgLink="/static/images/icon-m-user.svg"
-              text={
-                details ? (
-                  <Link href="/accounts">
-                    <a className="dashboard-link">
-                      {details.accountsCount.toLocaleString()}
-                    </a>
-                  </Link>
-                ) : (
-                  ""
-                )
-              }
-              loading={loading}
-            />
-          </Col>
-        </Row>
+                      <p>
+                        <a href="https://docs.near.org/docs/concepts/overview">
+                          {"Learn more about the key concepts"}
+                        </a>
+                      </p>
+                    </Term>
+                  }
+                  imgLink="/static/images/icon-m-height.svg"
+                  text={context.dashState.lastBlockHeight.toLocaleString()}
```

Comment:
Is it going to display "0" when the data has not been loaded yet? We should not do that. We need to display some neutral value (N/A / loading / "..." / "-" / some other loading indication)